File: test/resources/HelloProphecy/prophecy_scala/pipelines/gold_sales/code/src/main/scala/io/prophecy/pipelines/gold_sales/graph/SortByDateAndZip.scala
package io.prophecy.pipelines.gold_sales.graph

import io.prophecy.libs._
import io.prophecy.pipelines.gold_sales.udfs.PipelineInitCode._
import io.prophecy.pipelines.gold_sales.udfs.UDFs._
import io.prophecy.pipelines.gold_sales.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object SortByDateAndZip {

  def apply(context: Context, in: DataFrame): DataFrame =
    in.orderBy(col("order_date").asc, col("zipcode").asc)

}

File: test/resources/HelloProphecy/prophecy/pipelines/gold_sales/code/gold_sales/graph/SortByDateAndZip.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from gold_sales.config.ConfigStore import *
from gold_sales.udfs.UDFs import *

def SortByDateAndZip(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.orderBy(col("order_date").asc(), col("zipcode").asc())
